README.md

```diff
@@ -42,13 +42,16 @@ Before building Device Manager, the following steps are required:
   ```
 - Docker 20.10.18 (minimum version) installed,
 - Certificates delivered or generated and placed into (project root)/build/certs (create this folder if necessary) (scripts for generation can be found [here](https://github.com/ODIM-Project/ODIM/tree/main/build/cert_generator)),
-- Config changes to fit your needs:
+- Config changes to fit your needs(insert your own password in configs below - use SHA3-512 hash and encode it with base64):
```

Comment:
Password or passwords? Do we need the same password for user in redis schema and plugin? If not, maybe add a note, that the passwords can be the same, but not need to be.